pkg/syncer/metadatasyncer.go

```diff
@@ -411,20 +411,44 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 	volumeHealthTicker := time.NewTicker(time.Duration(getVolumeHealthIntervalInMin(ctx)) * time.Minute)
 	defer volumeHealthTicker.Stop()
 
+	// Trigger get pv to vvol mapping status.
 	// Trigger get volume health status.
-	if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorWorkload {
+	if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorWorkload || metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorVanilla {
 		go func() {
 			for ; true; <-volumeHealthTicker.C {
 				ctx, log = logger.GetNewContextWithLogger()
-				if !metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.VolumeHealth) {
-					log.Warnf("VolumeHealth feature is disabled on the cluster")
+				if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorVanilla {
+					if !metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.PvVvolMapping) {
+						log.Warnf("PV to vVol mapping feature is disabled on the cluster")
+					} else {
+						log.Infof("getVolumeHealthStatus is triggered")
+						csiGetVolumeHealthStatus(ctx, k8sClient, metadataSyncer, false)
+					}
 				} else {
-					log.Infof("getVolumeHealthStatus is triggered")
-					csiGetVolumeHealthStatus(ctx, k8sClient, metadataSyncer)
+					if !metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.VolumeHealth) {
+						log.Warnf("VolumeHealth feature is disabled on the cluster")
```

Comment:
Warnf->Info

Reply:
I think this part is from the original VolumeHealth code, so it is fine to stay as `Warnf`.